cmd/helm/registry_login.go

```diff
@@ -41,10 +41,11 @@ func newRegistryLoginCmd(cfg *action.Configuration, out io.Writer) *cobra.Comman
 	var passwordFromStdinOpt, insecureOpt bool
 
 	cmd := &cobra.Command{
-		Use:   "login [host]",
-		Short: "login to a registry",
-		Long:  registryLoginDesc,
-		Args:  require.MinimumNArgs(1),
+		Use:               "login [host]",
+		Short:             "login to a registry",
+		Long:              registryLoginDesc,
+		Args:              require.MinimumNArgs(1),
+		ValidArgsFunction: noCompletions,
```

Comment:
Must a registry start with "oci://" or could it be something else? If it MUST be "oci://" we could have the shell completion suggest it.

Reply:
I see that in https://helm.sh/docs/topics/registries/ there is an example that does `helm registry login -u myuser localhost:5000`.  So the `oci://` prefix is not necessary, so we shouldn't suggest it.